Dockerfile

```diff
@@ -1,9 +1,9 @@
 FROM golang:1.16
 
 # Install pact ruby standalone binaries
-RUN curl -LO https://github.com/pact-foundation/pact-ruby-standalone/releases/download/v1.37.0/pact-1.37.0-linux-x86_64.tar.gz; \
-    tar -C /usr/local -xzf pact-1.37.0-linux-x86_64.tar.gz; \
-    rm pact-1.37.0-linux-x86_64.tar.gz
+RUN curl -LO https://github.com/pact-foundation/pact-ruby-standalone/releases/download/v1.88.78/pact-1.88.78-linux-x86_64.tar.gz; \
+    tar -C /usr/local -xzf pact-1.88.78-linux-x86_64.tar.gz; \
+    rm pact-1.88.78-linux-x86_64.tar.gz
```

Comment:
Wasn't sure how necessary this was, but figured bumping the version to current which would add support for the branch tag wouldn't hurt.